PowerEditor/src/Parameters.cpp

```diff
@@ -405,6 +405,15 @@ static const WinMenuKeyDefinition winKeyDefs[] =
 
 	{ VK_F5,      IDM_EXECUTE,                                  false, false, false, nullptr },
 
+	{ VK_NULL,    IDM_WINDOW_SORT_FN_ASC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FN_DSC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FP_ASC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FP_DSC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FT_ASC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FT_DSC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FS_ASC,                       false, false, false, nullptr },
+	{ VK_NULL,    IDM_WINDOW_SORT_FS_DSC,                       false, false, false, nullptr },
+

```

Comment:
> changes are taking effect only after language change in preferences.

I guess I do understand what you said now - my bad, I forgot these entries.
I'll do a PR to fix them (and you will understand why this behaviour you describe above).

Reply:
Thank you.